### PR TITLE
Enable verbosity for Kube Linter by default

### PR DESCRIPTION
**Pull request by ljakimczuk**

### CODE REVIEW

File: app_build_suite/build_steps/helm.py [hunk 1]
```diff
@@ -298,8 +298,8 @@ def run(self, config: argparse.Namespace, _: Context) -> None:
             "lint",
             config.chart_dir,
         ]
-        if config.debug:
-            args.append("--verbose")
+
```
Comment (piontec) [inline app_build_suite/build_steps/helm.py:301]: Please add a comment why we unconditionally call `--verbose`, ok?

File: app_build_suite/build_steps/helm.py [hunk 2]
```diff
@@ -298,8 +298,8 @@ def run(self, config: argparse.Namespace, _: Context) -> None:
             "lint",
             config.chart_dir,
         ]
-        if config.debug:
-            args.append("--verbose")
+    
+        args.append("--verbose")
```
Comment (kubasobon) [inline app_build_suite/build_steps/helm.py:302]: Why not put it directly in the `args` list?

File: app_build_suite/build_steps/helm.py (hunk 1 above)
Comment (ljakimczuk) [reply app_build_suite/build_steps/helm.py]: good idea, done

File: app_build_suite/build_steps/helm.py (hunk 2 above)
Comment (ljakimczuk) [reply app_build_suite/build_steps/helm.py]: make sense, done

File: app_build_suite/build_steps/helm.py [hunk 3]
```diff
@@ -297,9 +297,13 @@ def run(self, config: argparse.Namespace, _: Context) -> None:
...
             "lint",
             config.chart_dir,
+            "--verbose",
         ]
-        if config.debug:
-            args.append("--verbose")
+
+        # adding `--verbose` to the args by default due to the kube-linter hidding
+        # some problems and thus skipping lintering some of the resources without informing
+        # the user.
```
Comment (rossf7) [inline app_build_suite/build_steps/helm.py:305]:
```suggestion
        # adding `--verbose` to the default args due to the kube-linter hiding
        # some problems and thus skipping linting some of the resources.
```

Some minor edits. WDYT? 

The linter is there to inform the user. So I think the last part is implied.

**Review by rossf7 — APPROVED** [review]

File: app_build_suite/build_steps/helm.py (hunk 3 above)
Comment (ljakimczuk) [reply app_build_suite/build_steps/helm.py]: sure! Thanks @rossf7

**Pull request merged**